src/me/mrCookieSlime/Slimefun/api/BlockStorage.java

```diff
@@ -665,4 +667,38 @@ public boolean hasUniversalInventory(Location l) {
 		String id = checkID(l);
 		return id == null ? false: hasUniversalInventory(id);
 	}
+
+	private void addNewInfo() {
+		File out = new File(path_blocks + world.getName() + "/CARGO_NODE_OUTPUT_ADVANCED.sfb");
+		File in = new File(path_blocks + world.getName() + "/CARGO_NODE_INPUT_ADVANCED.sfb");
+		FileConfiguration outcfg = YamlConfiguration.loadConfiguration(out);
+		FileConfiguration incfg = YamlConfiguration.loadConfiguration(in);
+		for(String key : outcfg.getKeys(false))
+			addFurnaceInfo(key);
+		for(String key : incfg.getKeys(false))
+			addFurnaceInfo(key);
+	}
+
+	private void addFurnaceInfo(String key) {
+		Block b = deserializeLocation(key).getBlock();
+		if(getAttachedBlock(b).getType().equals(Material.FURNACE) || getAttachedBlock(b).getType().equals(Material.BURNING_FURNACE)) {
```

Comment:
addNewInfo() does not seem to be called anywhere and could crash as it mass loads a ton of chunks without any throttling.